Docs/Architecture Decision Records/001 - PlainTextDeserializationVisitor.md

```diff
@@ -0,0 +1,22 @@
+# 1. Support "deserialization" of plain text 
+
+Date: 2020-05-07
+
+## Status
+
+Accepted
+
+## Context
+
+Some APIs, although they declare themselves as REST, return **a plain text string** as response. This breaks the serializer flow as it is expecting, by default, a JSON string. The result is a failure in deserialization and **the processor is interrupted**.
+
+## Decision
+
+We decided to implement a custom deserializer **that accepts plain text**. 
+
+A conditional before calling the serializer would have worked as well, but the resulting flow would have been confusing and complicated.
```

Comment:
On the scope of just core bundle, we can't be aware of this. We need to think on the bundles as standalone piece or including the dependencies these have, but not on the uses someone can do on that. In this case I would delete this sentence

Reply:
Gone